include/picongpu/particles/shapes/Counter.hpp

```diff
@@ -64,7 +79,7 @@ namespace shapes
             }
         };
 
-        struct ChargeAssignmentOnSupport : public shared_Counter::Counter
+        struct ChargeAssignmentOnSupport : public detail::Counter
```

Comment:
The comment for the operator is wrong: `[-support/2, support/2)` would mean `[-0/2, 0/2)`

Reply:
I think it's not the comment is wrong. But the whole shape is a workaround for something.